apps/meteor/server/lib/ldap/Connection.ts

```diff
@@ -349,9 +349,9 @@ export class LDAPConnection {
 
 				const entries: T[] = [];
 
-				res.on('searchEntry', (entry) => {
+				res.on('searchEntry', async (entry) => {
```

Comment:
ldapjs doesn't support async callbacks. Use `Promise.await` here and let's rework the ldap stuff on a separate PR.